cli/src/semdep/parsers/package_lock.py

```diff
@@ -22,6 +23,96 @@
 logger = getLogger(__name__)
 
 
+def parse_packages_field(deps: Dict[str, JSON]) -> List[FoundDependency]:
+    deps_json = deps.get("packages")
+    if deps_json is None:
+        logger.debug("Found package-lock with no 'packages'")
+        return []
+
+    deps = deps_json.as_dict()
```

Comment:
you're overwriting an existing variable, one that comes from the arguments. this is (i think) typically not best practice in python